tests/e2e/api_event_receiver_group_test.go

```diff
@@ -0,0 +1,234 @@
+			assert.Check(t, createdAt.Equal(updatedAt), "on init times should be the same")
+		})
+	}
+}
+
+func TestCreateInvalidGroup(t *testing.T) {
+	client := common.NewHTTPClient()
+
+	// common valid receiver used for mixing in with invalid inputs
+	receiverID, err := createReceiver(client, eventReceiverInput{
+		Name:        "common-group-receiver",
+		Type:        "my.receiver.type",
+		Version:     "0.0.3",
+		Description: "common receiver for invalid group testing",
+		Schema:      `{}`,
+	})
+	assert.NilError(t, err)
+
+	tests := map[string]struct {
+		input eventReceiverGroupInput
+	}{
+		"empty receiver id": {
+			input: eventReceiverGroupInput{
+				Name:        "my group",
+				Type:        "my.group",
+				Version:     "1.9.7",
+				Description: "my group",
+				Receivers:   []string{"", receiverID},
+			},
+		},
+		"non-existent receiver": {
+			input: eventReceiverGroupInput{
+				Name:        "your group",
+				Type:        "your.group",
+				Version:     "5.3.2",
+				Description: "your group",
+				Receivers:   []string{receiverID, "should-not-exist"},
+			},
+		},
+	}
+
+	for testName, tt := range tests {
+		t.Run(testName, func(t *testing.T) {
+			resp, err := client.Post(groupURI, "application/json", strings.NewReader(tt.input.toPayload()))
+			assert.NilError(t, err)
+			assert.Equal(t, resp.StatusCode, http.StatusBadRequest)
+
+			var body postGroupResponse
+			err = json.NewDecoder(resp.Body).Decode(&body)
+			assert.NilError(t, err)
+			assert.Check(t, len(body.Errors) > 0, "expect an error for invalid input")
+			assert.Equal(t, len(body.Data), 0, "expect no id generated")
+		})
+	}
+}
+
+func TestToggleGroup(t *testing.T) {
+	client := common.NewHTTPClient()
+
+	groupID, err := createGroup(client, eventReceiverGroupInput{
+		Name:        "group to be toggled",
+		Type:        "toggled.group",
+		Version:     "2.0.11",
+		Description: "group that will be toggled",
+		Receivers:   []string{},
+	})
+	assert.NilError(t, err)
+
+	originalGroup, err := getGroup(client, groupID)
+	assert.NilError(t, err)
+	originalUpdatedAt := time.Time(originalGroup.UpdatedAt.Date)
+	assert.Equal(t, originalGroup.Enabled, true, "group should be enabled by default")
+
+	toggleGroup(t, client, groupID, false)
+	group, err := getGroup(client, groupID)
+	assert.NilError(t, err)
+	assert.Equal(t, group.Enabled, false, "group should be disabled")
+	afterDisableUpdatedAt := time.Time(group.UpdatedAt.Date)
+	assert.Check(t, !afterDisableUpdatedAt.Equal(originalUpdatedAt), "updated time should be updated after disable")
+
+	toggleGroup(t, client, groupID, true)
+	group, err = getGroup(client, groupID)
+	assert.NilError(t, err)
+	assert.Equal(t, group.Enabled, true, "group should be enabled")
+	afterEnableUpdatedAt := time.Time(group.UpdatedAt.Date)
+	assert.Check(t, !afterEnableUpdatedAt.Equal(afterDisableUpdatedAt), "updated time should be updated after enable")
+}
+
+func toggleGroup(t testing.TB, client *http.Client, id string, enabled bool) {
```

Comment:
nit: I'd throw this in api_event_receiver_group.go. I think it makes more sense there.